src/calendar-list/index.js

```diff
@@ -187,7 +187,8 @@ class CalendarList extends Component {
 
   renderCalendar({item}) {
     return (
-      <CalendarListItem 
+      <CalendarListItem
+        calendarListContainer={this}
```

Comment:
Passing this/ref of the list to every item is not a good practice. You should use a callback which you then invoke in the onPress functions. For example: 
moveMonth = (month) => { 
    this.scrollToMonth(month); 
} 
and pass it to the item:  moveMonth={this.moveMonth}
and invoke it safely from the item: _.invoke(this.props, 'moveMonth');

Reply:
Ok, I will see what I can do